src/app.js

```diff
@@ -42,13 +42,14 @@ document.addEventListener('DOMContentLoaded', function () {
         deselectAll: function() { global.fileList.Select.deselectAll(); },
         selectUp: function(event) { global.fileList.Select.moveDirection('up', !event.shiftKey); },
         selectDown: function(event) { global.fileList.Select.moveDirection('down', !event.shiftKey); },
+        // removeSelected: function() { global.fileList.Select.removeSelected(); },
         handleCmdOrCtrlA: function(event) {
           if ((event.metaKey || e.ctrlKey) && event.keyCode === 65) {
             event.preventDefault();
             global.fileList.Select.selectAll();
           }
         },
-        handleCmdOrCtrlBackspace: function(event) {
+        handleCmdOrCtrlDelete: function(event) {
```

Comment:
It's still called backspace: https://css-tricks.com/snippets/javascript/javascript-keycodes/

Delete is `46`.